src/mermaid-isomorphic.ts

```diff
@@ -169,7 +170,7 @@ async function renderDiagrams({
     return result
   }
 
-  return Promise.allSettled(
+  return pSettle(
```

Comment:
This runs in the browser. We can’t use external functions here.

Reply:
sorry, this pull was opened by mistake